pull/context.go

```diff
@@ -27,6 +27,9 @@ type MembershipContext interface {
 
 	// IsOrgMember returns true if the user is a member of the given organzation.
 	IsOrgMember(org, user string) (bool, error)
+
+	// IsCollaborator returns true if the user meets the desiredPerm of the given organzation's repository.
+	IsCollaborator(desiredPerm, org, repo, user string) (bool, error)
```

Comment:
style: I'd put `desiredPerm` last for consistency with the `google/go-github` API style